.github/workflows/dt-test-and-report-code-coverage.yml

```
@@ -26,8 +26,12 @@ jobs:
           egress-policy: audit
 
       - id: get_url
+        env:
+          REPOSITORY: ${{ github.repository }}
+          WORKFLOW: ${{ github.workflow }}
+          RUN_ID: ${{ github.run_id }}
         run: |
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)